src/uncapitalize.ts

```diff
@@ -0,0 +1,47 @@
+import { purry } from "./purry";
+
+/**
+ * Makes first character of a string lower-case. Uses the built-in
+ * [`String.prototype.toLowerCase`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase)
+ * for the runtime, and the built-in [`Uncapitalize`](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html#Uncapitalizestringtype)
+ * utility type for typing.
+ *
+ * For other case manipulations see: `toUpperCase`, `toLowerCase`, `capitalize`,
+ * and `toCamelCase`.
+ *
+ * @param data - A string.
+ * @signature
+ *   R.uncapitalize(data);
+ * @example
+ *   R.uncapitalize("HELLO WORLD"); // "hELLO WORLD"
+ * @dataFirst
+ * @category String
+ */
+export function uncapitalize<T extends string>(data: T): Uncapitalize<T>;
+
+/**
+ * Makes first character of a string upper-case. Uses the built-in
+ * [`String.prototype.toLowerCase`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase)
+ * for the runtime, and the built-in [`Uncapitalize`](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html#Uncapitalizestringtype)
+ * utility type for typing.
+ *
+ * For other case manipulations see: `toUpperCase`, `toLowerCase`, `capitalize`,
+ * and `toCamelCase`.
+ *
```

Comment:
**question:** why not a similar note about non-ASCII characters?

Reply:
There's no "text-transform" value that would uncapitalize, so there was no alternative to offer to users.